Raven.Database/Raft/Controllers/ClusterController.cs

```diff
@@ -144,6 +147,16 @@ public Task<HttpResponseMessage> CanInstallSnapshot([FromUri]CanInstallSnapshotR
             return taskCompletionSource.Task;
         }
 
+
+        [HttpPost]
+        [RavenRoute("cluster/replicationState")]
+        public async Task<HttpResponseMessage> ReplicationState()
```

Comment:
This is how other commands were implemented, i was just keeping it inline with the other commands